### PR TITLE
Update package management to go mod

### PR DESCRIPTION
**Pull request by AntonParaskiv**
…ubesim

* Added go mod files
* Updated vendor

### CODE REVIEW

**Comment by AntonParaskiv** [comment]
Hi!
This is solution of issue #7

**Comment by k82cn** [comment]
@AntonParaskiv , prefer to keep package to `volcano.sh/kubesim`; similar to `volcano.sh/volcano` :)

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+	github.com/stretchr/testify v1.6.2-0.20200628121210-87a988cffbb9
+	google.golang.org/grpc v1.26.0
+	gopkg.in/yaml.v3 v3.0.0-20200615113413-eeeca48fe776 // indirect
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v0.0.0
+	k8s.io/component-base v0.0.0
+	k8s.io/cri-api v0.0.0
+	k8s.io/klog v1.0.0
+	k8s.io/kubernetes v0.0.0-20200630205207-e38139724f8f
```
Comment (k82cn) [inline go.mod]: let's add version to k8s related repo :)

Comment (AntonParaskiv) [reply go.mod]: > let's add version to k8s related repo :)

there is no tag version for commit e38139724f8f0086be9db575d8fe5211c653dc8f, that pointed in README.md
nearest tagged commit is dff82dc0de47299ab66c83c626e08b245ab19037 v1.18.6
should am i use that?

Comment (k82cn) [reply go.mod]: `v1.18.6` is ok to me :) please also update other related repo versions :)

File: cmd/kubesim/hollow-node.go [hunk 2]
```diff
@@ -45,16 +45,16 @@ import (
...
-	"volcano.sh/kubesim/pkg/kubesim"
-	"volcano.sh/kubesim/pkg/metrics"
-	metricconfig "volcano.sh/kubesim/pkg/metrics/config"
-	cadvisortest "volcano.sh/kubesim/pkg/mock/kubelet/cadvisor/testing"
-	"volcano.sh/kubesim/pkg/mock/kubelet/remote"
-	fakeremote "volcano.sh/kubesim/pkg/mock/kubelet/remote/fake"
-	fakeexec "volcano.sh/kubesim/pkg/mock/util/exec/testing"
-	fakeiptables "volcano.sh/kubesim/pkg/mock/util/iptables/testing"
-	fakesysctl "volcano.sh/kubesim/pkg/mock/util/sysctl/testing"
+	simulatorconfig "github.com/volcano-sh/kubesim/pkg/config"
```
Comment (k82cn) [inline cmd/kubesim/hollow-node.go]: let's keep `volcano.sh` org :)

**Comment by k82cn** [comment]
/lgtm
/approve

Thanks very much for your contribution!!

**Pull request merged**